### PR TITLE
versionId support in presigned url

### PR DESCRIPTION
**Pull request by prakashsvmx**
VersionId support in pre signed url ( get)

### CODE REVIEW

**Review by kannappanr — DISMISSED** [review]
LGTM

File: examples/presigned-getobject-request-date.js [hunk 1]
```diff
@@ -27,8 +27,14 @@ var s3Client = new Minio.Client({
 })
 
 // Presigned get object URL for my-objectname at my-bucketname, it expires in 7 days by default.
-var requestDate = new Date('2017-01-01');
-var presignedUrl = s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {}, requestDate, function(e, presignedUrl) {
+var requestDate = new Date('2017-01-01')
```
Comment (ebozduman) [inline examples/presigned-getobject-request-date.js:30]: We don't need this old date. We can set `requestDate` to current date
```suggestion
var requestDate = new Date()
```

File: examples/presigned-getobject-request-date.js [hunk 2]
```diff
@@ -27,8 +27,14 @@ var s3Client = new Minio.Client({
...
+s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {}, requestDate, function(e, presignedUrl) {
   if (e) return console.log(e)
   console.log(presignedUrl)
 })
+
+// Versioning support
+s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {versionId:"10fa9946-3f64-4137-a58f-888065c0732e"}, new Date('2017-01-01'), function(e, presignedUrl) {
```
Comment (ebozduman) [inline examples/presigned-getobject-request-date.js:37]: The same comment as before:
```suggestion
s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {versionId:"10fa9946-3f64-4137-a58f-888065c0732e"}, new Date(), function(e, presignedUrl) {
```

**Review by ebozduman — CHANGES_REQUESTED** [review]

**Comment by prakashsvmx** [comment]
@ebozduman  Thank you for the review. I have addressed the comments. PTAL

**Review by ebozduman — APPROVED** [review]
LGTM

**Comment by ebozduman** [comment]
Oops!
@prakashsvmx 
Please take a look into the build failures

**Comment by kannappanr** [comment]
@prakashsvmx Please resolve the conflicts

**Review by ebozduman — DISMISSED** [review]
LGTM

**Review by kannappanr — APPROVED** [review]
LGTM

**Pull request merged**